### DISCUSSION TITLE
Собираем ошибки в распознавании названий смартапов

### DISCUSSION THREAD

**Discussion by oblomil** [sberdevices/salute-issues#179] [ASR, TTS]
Наши ассистенты каждый день учатся чему-то новому. Одна из важнейших задач ассистентов – быстро запустить нужный пользователю смартап. Часто бывает, что название смартапа – креативное, но не очень распространенное в русском языке слово или выражение.

**Вы можете помочь нам с более точной настройкой сервиса распознавания речи, поделившись примером ошибочного или неудачного распознавания названия (или просто каких-то слов) в вашем смартапе.**

Коллекционируем такие случаи в этом тикете. 

Спасибо за участие!

**Comment by subnetsRU** [comment]
"моя фоторамка" часто распознаётся как "моя футурама"
"крути барабан" часто распознаётся как "крутой барабан"

**Comment by danchistyakov** [comment]
Плохо распознается фраза "Запусти подготовку к ОГЭ по обществознанию".
Возможные варианты: "Подготовку Куган по обществознанию";
"Подготовку кого по обществознанию";
"Подготовку угол по обществознанию";
"Подготовку ОГЭ по обществознанию";
"Подготовку Google по обществознанию";
"Подготовку Гугль по обществознанию";
"Подготовку кугл по обществознанию";
"Подготовку Гуга по обществознанию";
"Подготовку kygo по обществознанию".

**Comment by danchistyakov** [comment]
Плохо распознается фраза "Запусти подготовку к ОГЭ по обществознанию".
Возможные варианты: "Подготовку Куган по обществознанию";
"Подготовку кого по обществознанию";
"Подготовку угол по обществознанию";
"Подготовку ОГЭ по обществознанию";
"Подготовку Google по обществознанию";
"Подготовку Гугль по обществознанию";
"Подготовку кугл по обществознанию";
"Подготовку Гуга по обществознанию";
"Подготовку kygo по обществознанию".
"Подготовку как по обществознанию".

**Comment by Sergeycul** [comment]
"Музыкальные стулья" в 90% распознается, как "Музыкальная история" и включает мне песни.
Конечно, песни - приятно, но не ТО ))

**Comment by snyuryev** [comment]
IT (ай-ти) - всегда распознает как "найти".
